Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,24 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testEncodePropertyListEncoder() {
+        let point = CGPoint(x: 1.1, y: 2.2)
+        let temporaryDirectoryURL = FileManager.default.temporaryDirectory
+        let temporaryFilename = ProcessInfo().globallyUniqueString
+        let fileURL = temporaryDirectoryURL.appendingPathComponent(temporaryFilename)
+
+        XCTAssertNoThrow(try FileManager.default.encode(point, to: fileURL));
```

Comment:
Use `do-catch` like with the JSON extensions. It makes more sense, and then you can remove the `NoThrow` assertions and check the values (where necessary).

Reply:
fixed